src/main/java/edu/kis/powp/jobs2d/drivers/RecordMacroDriver.java

```diff
@@ -0,0 +1,33 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.features.MacroFeature;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+
+public class RecordMacroDriver implements Job2dDriver{
+    private final Job2dDriver job2dDriver;
```

Comment:
RecordMacroDriver -> RecordMacroDriverDecorator
PR #14 introduced DriverDecorators. Please check it out and use it.